tensorflow/python/eager/benchmarks_test.py

```diff
@@ -1620,5 +1627,46 @@ def benchmark_tf_range_return_int32_GPU(self):
   def benchmark_tf_range_return_int64_GPU(self):
     self._benchmark_tf_range_return(dtype=dtypes.int64, device=GPU)
 
+  def benchmark_embedding_lookup_sparse_with_sparse_input(self,
```

Comment:
Is it possible to add some benchmark tests results for allow_dense_grad=False vs True? Also add the results to description. I feel that will be useful to justify the public API change.